src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java

```diff
@@ -144,7 +151,47 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> consumer)
 		CombinationRecipeBuilder.of(Items.MOSSY_COBBLESTONE_WALL).input(Items.COBBLESTONE_WALL).or().namedInput(Items.WHEAT_SEEDS).buildFor(consumer, Minestuck.MOD_ID);
 		CombinationRecipeBuilder.of(Items.MOSSY_COBBLESTONE_WALL).input(Items.COBBLESTONE_WALL).or().namedInput(Items.MOSSY_COBBLESTONE).buildFor(consumer, Minestuck.MOD_ID);
 		CombinationRecipeBuilder.of(Items.MOSSY_STONE_BRICKS).input(Items.STONE_BRICKS).or().input(Items.WHEAT_SEEDS).buildFor(consumer, Minestuck.MOD_ID);
-		CombinationRecipeBuilder.of(Items.CLAY).input(Items.SAND).and().input(Items.WATER_BUCKET).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.AMETHYST_CLUSTER).input(Items.GRASS_BLOCK).and().input(Items.AMETHYST_SHARD).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.LARGE_AMETHYST_BUD).input(Items.SPYGLASS).and().input(Items.MEDIUM_AMETHYST_BUD).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.MEDIUM_AMETHYST_BUD).input(Items.SPYGLASS).and().input(Items.AMETHYST_SHARD).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.SMALL_AMETHYST_BUD).input(Items.SPYGLASS).or().input(Items.MEDIUM_AMETHYST_BUD).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.CALCITE).input(Items.AMETHYST_SHARD).or().input(Items.STONE).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.CANDLE).input(Items.BOOK).and().input(Items.TORCH).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.COBBLED_DEEPSLATE).input(Items.COBBLESTONE).and().input(Items.CHARCOAL).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.DEEPSLATE).input(Items.STONE).or().input(Items.INK_SAC).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.CHAINMAIL_BOOTS).and().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.DIAMOND_BOOTS).or().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.IRON_BOOTS).and().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.NETHERITE_BOOTS).or().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.LEATHER_BOOTS).and().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.GOLDEN_BOOTS).or().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
+		CombinationRecipeBuilder.of(Items.POINTED_DRIPSTONE).input(Items.DRIPSTONE_BLOCK).or().input(Items.ARROW).buildFor(consumer, Minestuck.MOD_ID);
```

Comment:
the spyglass was purposeful. i thought it would be interesting to present alchemy as more a combination of ideas as it is in the comic. in this case, the spyglass is magnifying or shrinking a given object

Reply:
If you would like an example of how the prismarine armor is coded, I can give you an example you can utilize